lib/Dialect/ESI/runtime/python/esiaccel/cosim/questa.py

```diff
@@ -53,9 +97,28 @@ def run_command(self, gui: bool) -> List[str]:
           vsim,
           "driver_opt",
           "-batch",
+      ]
+
+      if self.debug:

```

Comment:
FYI: I also added the capability to dump a VCD using the SAVE_WAVE functionality just like in Verilator.